templatekit/filerender.py

```diff
@@ -15,7 +15,7 @@
 from jinja2.exceptions import TemplateSyntaxError
 
 
-def render_file_template(template_path):
+def render_file_template(template_path, use_defaults=False):
```

Comment:
Why did you switch the default to `False`?  Before a user couldn't choose `False`.

Reply:
Yeah, I think a commit message covers this. tl;dr is that the module was initially designed just for the scons example building case, but I actually want that to now be the edge case. The default is that someone uses the templates to generate a non-default project.